packages/perseus/src/renderer.tsx

```diff
@@ -161,34 +161,32 @@ export type Widget = {
     examples?: () => ReadonlyArray<string>;
 };
 
-type Props = Partial<React.ContextType<typeof DependenciesContext>> & {
-    apiOptions?: APIOptions;
-    alwaysUpdate?: boolean;
-    findExternalWidgets: any;
-    highlightedWidgets?: ReadonlyArray<any>;
-    images: PerseusRenderer["images"];
-    keypadElement?: any; // TODO(kevinb): add proper types,
-    onInteractWithWidget: (id: string) => void;
-    onRender: (node?: any) => void;
-    problemNum?: number;
-    questionCompleted?: boolean;
-    reviewMode?: boolean | null | undefined;
-    content: PerseusRenderer["content"];
-    serializedState?: any;
-    // Callback which is called when serialized state changes with the new
-    // serialized state.
-    onSerializedStateUpdated: (serializedState: {
-        [key: string]: any;
-    }) => unknown;
-    // If linterContext.highlightLint is true, then content will be passed
-    // to the linter and any warnings will be highlighted in the rendered
-    // output.
-    linterContext: LinterContextProps;
-    legacyPerseusLint?: ReadonlyArray<string>;
-    widgets: PerseusRenderer["widgets"];
-    // Skip adding paragraph class
-    inline?: boolean;
-};
+type Props = Partial<React.ContextType<typeof DependenciesContext>> &
+    Pick<PerseusRenderer, "content" | "images" | "widgets"> & {
+        apiOptions?: APIOptions;
+        alwaysUpdate?: boolean;
+        findExternalWidgets: any;
+        highlightedWidgets?: ReadonlyArray<any>;
+        keypadElement?: any; // TODO(kevinb): add proper types,
+        onInteractWithWidget: (id: string) => void;
+        onRender: (node?: any) => void;
+        problemNum?: number;
+        questionCompleted?: boolean;
+        reviewMode?: boolean | null | undefined;
+        serializedState?: any;
+        // Callback which is called when serialized state changes with the new
+        // serialized state.
+        onSerializedStateUpdated: (serializedState: {
+            [key: string]: any;
+        }) => unknown;
+        // If linterContext.highlightLint is true, then content will be passed
+        // to the linter and any warnings will be highlighted in the rendered
+        // output.
+        linterContext: LinterContextProps;
+        legacyPerseusLint?: ReadonlyArray<string>;
+        // Skip adding paragraph class
+        inline?: boolean;
+    };
```

Comment:
Why did this type change? Looks like we lost some properties (images, content, and widgets).

Reply:
I reverted this.  I had rebased on top of `main` and incorrectly done a conflict resolution. Thanks for catching this!